frontend/apps/app/libs/utils/dateFormat.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Format date to "MMM DD, YYYY" format (e.g., "Jan 15, 2025")
+ * Uses consistent en-US locale to prevent server/client hydration mismatches
+ */
+export const formatDate = (dateString: string): string => {
+  const date = new Date(dateString)
+  return date.toLocaleDateString('en-US', {
+    month: 'short',
+    day: 'numeric',
+    year: 'numeric',
+  })
+}
+
+/**
+ * Format date to "M/D/YYYY" format (e.g., "1/15/2025")
+ * Uses consistent en-US locale to prevent server/client hydration mismatches
+ */
+export const formatDateShort = (dateString: string): string => {
+  const date = new Date(dateString)
+  return date.toLocaleDateString('en-US', {
+    month: 'numeric',
+    day: 'numeric',
+    year: 'numeric',
+  })
+}
```

Comment:
I considered switching to ISO 8601, but decided to keep the current format for the sake of consistency.

**Reasons to keep the current `formatDateShort` (M/D/YYYY):**

1. Usability: For our target users (likely U.S.-based), a date like `8/12/2025` is more intuitive.
2. **Consistency**: Elsewhere we use the `Jan 15, 2025` format, which is also U.S.-style.

**When ISO 8601 is more appropriate:**

* API responses
* Database storage
* Log files
* Developer-facing tools

Since “Recent Sessions” is part of the end-user UI, prioritizing readability makes it more appropriate to stick with the current `formatDateShort`.